src/get-started/timestamping-tutorial.md

```diff
@@ -70,35 +80,30 @@ where we indicate the external libraries required for our service.
 ??? note "Imports"
 
     ```rust
-    extern crate chrono;
     #[macro_use]
-    extern crate exonum;
-    extern crate exonum_time;
+    extern crate exonum_derive;
     #[macro_use]
     extern crate failure;
     #[macro_use]
     extern crate log;
-    extern crate serde;
     #[macro_use]
     extern crate serde_derive;
-    extern crate serde_json;
 
     pub mod api;
+    pub mod proto;
     pub mod schema;
     pub mod transactions;
```

Comment:
Seems, that we may remove `mod` section from this snippet since we mention about `mods` further.